### PR TITLE
Remove more unused stuff / duplication

### PR DESCRIPTION
**Pull request by jessieay**
**WHY**: why not?

This was inspired by our low test coverage score in code climate. I
removed stuff we do not use / have tests for

### CODE REVIEW

File: app/controllers/address_controller.rb [hunk 1]
```diff
@@ -1,14 +1,4 @@
 # frozen_string_literal: true
 
 class AddressController < StandardStepsController
```
Comment (adarsh) [inline app/controllers/address_controller.rb]: Do we need to keep this file?

File: app/controllers/sign_and_submit_controller.rb [hunk 2]
```diff
@@ -1,14 +1,4 @@
 # frozen_string_literal: true
 
 class SignAndSubmitController < StandardStepsController
```
Comment (adarsh) [inline app/controllers/sign_and_submit_controller.rb]: Same, do we need to keep this still?

File: app/controllers/address_controller.rb (hunk 1 above)
Comment (jessieay) [inline app/controllers/address_controller.rb]: Yes we are using this to collect address info, but `update` is defined in ` StandardStepsController` so we don't need to duplicate the logic here

File: app/controllers/sign_and_submit_controller.rb (hunk 2 above)
Comment (jessieay) [inline app/controllers/sign_and_submit_controller.rb]: same as above! We are using it, but `update` does not have special logic so we can fall back on `StandardStepsController` -- thanks citizen code, for that!

**Pull request merged**